src/EditorFeatures/Test2/Workspaces/SymbolDescriptionServiceTests.vb

```diff
@@ -145,6 +145,86 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
 </Workspace>
             Await TestCSharpAsync(workspace, "class System.String")
         End Function
+      
+        <Fact>
+        Public Async Function TestCSharpStaticField() As Task
+            Dim workspace =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+            class Foo
+            {
+                private static int $$x
```

Comment:
💡 Unless the intent of this test is to cover an error case (incomplete declaration), it would be good to fix the syntax error.

```suggestion
                private static int $$x;
```

Reply:
Apologies, missed that in the merge. Thanks for catching that!